### ISSUE TITLE
feature: blacklist files for template helpers

### ISSUE THREAD

**Issue by srtucker22** [Urigo/tortilla#37]
e.g. If I'm using `yarn`, every time I add a package in a step, it updates `yarn.lock` which butchers {{{diffStep X}}} without the file declaration. No one needs to see the changes to the lock file, so it would be nice to just blacklist it somewhere (maybe like in a config file or `package.json` as suggested in #35)

**Comment by DAB0mB** [comment]
I think it's a good this to put all the necessary features in list and prioritize everything. There can be lots of features that should be implemented but we have limited resources, therefore we can't do everything. What do you think are the most important things we should focus on in the near future? Is this one of them?

**Comment by srtucker22** [comment]
this isn't one of them per se, but I think defining and including a basic configuration setup would be very important for housing features like this and in #35. As more features get requested/become important, I think many would naturally flow through that sort of setup.

That's still a feature request though. I personally think the most important issues are #33, #23, and breaking things when you don't add a msg to `step tag` or `step push` because they cause the most friction for new users like me.

**Comment by srtucker22** [comment]
I'm also thinking it would be helpful to make a tortilla tutorial on how to use tortilla `tortillaception`? Thinking about taking this on once I finish building my first tut.

**Comment by Urigo** [comment]
@DAB0mB in order to organize all issues and prioritize them you can use Github projects - https://github.com/Urigo/tortilla/projects

**Comment by DAB0mB** [comment]
@srtucker22 I agree, Tortilla needs a tutorial, but it shouldn't be something robust because the API might change, Tortilla is in alpha after all
